src/customer/strategies/masterpass/masterpass-customer-strategy.spec.ts

```diff
@@ -57,6 +58,8 @@ describe('MasterpassCustomerStrategy', () => {
 
         requestSender = createRequestSender();
 
+        locale = 'en-US';
```

Comment:
Same here, instead of assigning to a temporary variable, you could directly pass it to the function